gateway/1.4/modules/ROOT/pages/flex-view-replica-status.adoc

```diff
@@ -5,21 +5,45 @@ endif::[]
 :page-aliases: runtime-manager::flex-gateway-replica-status.adoc
 :imagesdir: ../assets/images
 
-To view information about the status of Flex Replicas:
+In Runtime Manager, you can view the status of groups of Flex Gateway replicas and individual replicas.
+
+== Groups of Flex Gateway Replicas
+
+To view information about the status of groups of Flex Gateway replicas:
 
 . From Anypoint Platform, select *Runtime Manager*.
-. Select *Flex Gateway* in the navigation menu.
+. Select *Flex Gateway* in the navigation menu. +
+By default, all Flex Gateways are shown. To filter by status, click the *Any Status* arrow and the status you want to filter.
+
+The *Name* column in the list of Flex Gateways lists the name and the number of connected replicas.
 
-The *Status* column in the list of gateways shows the connection status for the replicas associated with a particular gateway.
+The *Status* column shows the connection status for the replicas associated with a particular gateway.
+
+The following table describes the meaning of each status for a group of replicas:
+
+[%header,cols="30a,70a"]
+|===
+|Status | Group of Replicas
+|Connected | At least one replicas is to the platform and are serving traffic.
+|Disconnected | There are no replicas connected to the platform and no information on whether they are serving traffic.
+|===
+
+== Individual Replicas
+
+To view information about the status of groups of individual replicas:
+
+. From Anypoint Platform, select *Runtime Manager*.
+. Select *Flex Gateway* in the navigation menu.
+. Select the name of the Flex Gateway whose individual replica statuses you want to view. +
+By default, disconnected replicas are not shown. To view disconnected replicas, select *Show disconnected Replicas*. 
```

Comment:
Is "disconnected" not capitalized in the "Show disconnected Replicas" UI element?

Reply:
Not currently, I need to mention this to Tito though